tasks/lib/packages/installed_packages.js

```diff
@@ -1,7 +1,37 @@
-import { relative } from 'path';
+import { relative, resolve } from 'path';
+import { readFileSync } from 'fs';
 
 import { callLicenseChecker } from './license_checker';
 
+function resolveLicense(licenseInfo, key, licenseOverrides) {
+  const {
+    private: isPrivate,
+    licenses: detectedLicenses,
+    realPath,
+  } = licenseInfo[key];
+
+  // `license-checker` marks all packages that have `private: true`
+  // in their `package.json` as "UNLICENSED", so we try to lookup the
+  // actual license by reading the license field from their package.json
+  if (isPrivate && detectedLicenses === 'UNLICENSED') {
+    try {
+      const pkg = JSON.parse(readFileSync(resolve(realPath, 'package.json')));
+      if (!pkg.license) {
+        throw new Error('no license field');
```

Comment:
Every time I make modifications to this module I feel like we should just rewrite the license checker to not depend on https://github.com/davglass/license-checker